MD/md-15/README.md

```diff
@@ -0,0 +1,61 @@
+# MD-15: Movement Glossary
+- **Description**: Provide a glossary for Movement Vocabulary.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+
+
+## Overview
+In order to communicate with more specificity in all contexts, Movement Labs should introduce a glossary. This glossary should define terms both unique to Movement Labs and general terms that are used in a unique way, underspecified in the literature, or have a specific meaning in the context of Movement Labs.
+
+## Desiderata
+
+<!--
+  List out the specific desiderata. Each entry should consist of:
+
+  1. Title: A concise name for the desideratum.
+  2. User Journey: A one or two-sentence statement focusing on the "user" (could be a human, machine, software, etc.) and their interaction or experience.
+  3. Description (optional): A more detailed explanation if needed.
+  4. Justification: The reasoning behind the desideratum. Why is it necessary or desired?
+  5. Recommendations (optional): Suggestions or guidance related to the desideratum.
+
+  Format as:
+
+  ### Desideratum Title
+
+  **User Journey**: [user] can [action].
+
+  **Description**: <More detailed explanation if needed (optional)>
+
+  **Justification**: <Why this is a significant or required desideratum>
+
+  **Recommendations**: <Any specific guidance or suggestions (optional)>
+
+  TODO: Remove this comment before finalizing.
+-->
+### D1: Have a Common Format for a Glossary Entry (Gloss)
+
+**Justification**: A common format for glossary entries will make it easier to maintain and update the glossary.
+
+**Recommendations**: 
+- Create a new formatted document for glossary entries in the MIP repository. Call this document an MG, short for Movement Gloss.
+- Include sections for (1) term, (2) authorship, (3) concise definition, (4) catalog of references to definitions or usages of the term in other contexts, (5) example usages, and (6) errata.
+- Encourage formalism where applicable.
+
+### D2: Introduce Glossary Terms via an MIP
+
+**Justification**: Introducing glossary terms via an MIP will allow for a more formal process for introducing new terms.
```

Comment:
```suggestion
**Justification**: Introducing glossary terms via an MIP or MD will allow for a more formal process for introducing new terms.
```

Reply:
There is no information at the root README. i have [opened a PR](https://github.com/movementlabsxyz/MIP/pull/22) for this